src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -60,6 +61,7 @@ export default Vue.extend({
       isUpcoming: false,
       isPremium: false,
       hideViews: false,
+      channelBlocked: false,
```

Comment:
Channel blocking logic should not be handled by `ft-list-video`.  It should be handled by a wrapper component. Most of the time this wrapper is `ft-element-list`

Reply:
Recommendation doesn't use `ft-list-element` (I think this is the only case). Should I copy functionalities from `ft-list-element` to `Watch.js` ?